lib/confluence/aggressive_removal.es6.js

```diff
@@ -66,58 +67,70 @@ foam.CLASS({
             if (prevReleaseBrowsers.length < 2)
               return Promise.resolve(this.ArraySink.create());
 
-            let browser = prevReleaseBrowsers.pop();
+            // TODO(markdittmer): This assumes that browsers are in
+            // releaseDate-order. That assumption doesn't appear to be
+            // documented or enforced.
+            let latestPrevReleaseBrowser = prevReleaseBrowsers.pop();
             let removedIfaces = {};
             let promises = [];
-            // Find all APIs available this browser before date - 1yr.
+            // Find all APIs available this browser before date -1yr.
             for (let i = 0; i < prevReleaseBrowsers.length; i++) {
-              promises.push(prevReleaseBrowsers[i].interfaces.select({
-                put: function(_, iface) {
-                  if (!removedIfaces.hasOwnProperty(iface.interfaceKey)) {
-                    removedIfaces[iface.interfaceKey] = 0;
-                  }
-                },
-              }));
+              promises.push(prevReleaseBrowsers[i].interfaces.select(
+                  this.AnonymousSink.create({sink: {put: function(_, iface) {
+                    if (!removedIfaces.hasOwnProperty(iface.interfaceKey)) {
```

Comment:
You don't need the check. Setting it to 0 a whole bunch of times should be safe.